build/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM golang:1.10.4-alpine as build
+
+ARG PROJECT="go-api-template"
+
+RUN apk update && apk upgrade && apk add --no-cache bash git curl
```

Comment:
installing curl just do download dep seems like an overkill. We could just wget it.

Reply:
Also bash ? is sh not good enough?

Reply:
Probably leftovers when I did image debugging